content/hardware/03.nano/boards/nano-esp32/tutorials/debugging/debugging.md

```diff
@@ -9,6 +9,8 @@ tags: [ESP32, Debugging, IDE]
 
 Debugging skills are valuable not only for tackling complex projects but also for beginners and those working on intermediate-level projects, as they offer valuable insights into the specific behavior of your code.
 
+***Some boards from the first limited production batch were assembled with a different RGB LED which has the green and blue pins inverted. For more information read our full Help Center article [here](https://support.arduino.cc/hc/en-us/articles/9589073738012).***
```

Comment:
I'm not sure how I feel about this note being up here, as a reader I think it makes it seem as a note that as automatically tagged onto all nano esp32 tutorials and may not apply to this specific one, whereas putting it close to where LED behaviour is described ties it in better.

An alternative would be to add another note further down in the doc to say "hey remember the inverted LED thing???" where LED behaviour is described.